### DISCUSSION TITLE
Reset Button

### DISCUSSION THREAD

**Discussion by ukaeufe** [open-election-compass/client#45] [Q&A]
Hi,

I check if you tool can be used for election 2021.
I played around and found it very nice and easy to use.
In one demo I see a reset button.
This I cannot configure in your tool. Has that been manually added to the html?

Best regards

Uwe

**Comment by tillsanders** [comment]
Hi Uwe,

thank you for your feedback! For what project would you like to use it?

What you see in the demo is called the 'Kiosk Mode'. This was designed for situations where your election compass is running on a device located in a public place, e.g. a tablet in the market square for passers-by. In this mode, the election compass will ask to be reset after use or do so automatically after a period of inactivity, so the next person can start from the beginning. 

You can activate the Kiosk Mode in the [HTML Generator](https://open-election-compass.com/configurator/version-1/html-generator.html):

![Screenshot of the HTML Generator](https://user-images.githubusercontent.com/1533846/123400598-3a0ab600-d5a6-11eb-8d09-b77ab08a7c7f.png)

**Reply by ukaeufe** [reply]
I would use the tool for a VAA with question from the domain of animal welfare and rights. Thx for the answer. I will try and see if that meets my idea.
